code/modules/admin/verbs/modifyvariables.dm

```diff
@@ -381,10 +381,9 @@ var/list/VVckey_edit = list("key", "ckey")
 	if(!check_rights(R_VAREDIT))
 		return
 
-	for(var/p in forbidden_varedit_object_types)
-		if( istype(O,p) )
-			usr << "<span class='danger'>It is forbidden to edit this object's variables.</span>"
-			return
+	if(is_type_in_list(O, forbidden_varedit_object_types))

```

Comment:
Forbidden objects are admin ranks and holders and stuff like that, so that admins can't VV themselves all the permissions.